lib/src/bin/settings.yaml

```diff
@@ -1,16 +1,10 @@
 ---
 connection:
-  uri: "https://api.phylum.io"
+  uri: "https://api.staging.phylum.io"
 auth_info:
-  oidc_discovery_url: "https://login.phylum.io/auth/realms/phylum/.well-known/openid-configuration"
+  oidc_discovery_url: "https://login.staging.phylum.io/auth/realms/phylum/.well-known/openid-configuration"
+  offline_access: eyJhbGciOiJIUzI1NiIsInR5cCIgOiAiSldUIiwia2lkIiA6ICJhNjA2YWU1Ny02NWFiLTRjMGQtODBlZC01ZWExMzdmMGQ1ODIifQ.eyJleHAiOjE2NDI1NTMyNDEsImlhdCI6MTY0MTk0ODQ0MSwianRpIjoiMWM1MzM3YTUtOTEwZi00NjRmLTlhYmUtY2QzNWMzYTZkNWVmIiwiaXNzIjoiaHR0cHM6Ly9sb2dpbi5zdGFnaW5nLnBoeWx1bS5pby9hdXRoL3JlYWxtcy9waHlsdW0iLCJhdWQiOiJodHRwczovL2xvZ2luLnN0YWdpbmcucGh5bHVtLmlvL2F1dGgvcmVhbG1zL3BoeWx1bSIsInN1YiI6ImE3YTVkMjNiLTM3MDItNDNhZS04OWZlLWJhYTA5YjhmZmYzNiIsInR5cCI6Ik9mZmxpbmUiLCJhenAiOiJwaHlsdW1fY2xpIiwic2Vzc2lvbl9zdGF0ZSI6IjJkMzQxOGYxLWYzMjEtNGRkNi1hZDMxLTg3YTcyZGY0YmY4ZCIsInNjb3BlIjoib3BlbmlkIGVtYWlsIG9mZmxpbmVfYWNjZXNzIHByb2ZpbGUiLCJzaWQiOiIyZDM0MThmMS1mMzIxLTRkZDYtYWQzMS04N2E3MmRmNGJmOGQifQ.ix3RMoQzZBdD4xRAbjKDhk_eZ5BroG9a_1J2t7afJ9o
```

Comment:
Is this your token? Just checking if that should be added?

Reply:
It's a temp token, but I'll get that pulled.. :P

Swore I didn't add it.